src/VisualStudio/Core/Def/Implementation/Diagnostics/DiagnosticProgressReporter.cs

```diff
@@ -154,35 +155,30 @@ private void UpdateUI(ProgressData progressData)
             // Start the task center task if not already running.
             if (_taskHandler == null)
             {
-                StartTaskCenter();
+                // Make sure to stop the previous task center task if present.
+                StopTaskCenter();
```

Comment:
note: not actually necessary :)    `_taskHandler == null` *is* the state where there is nothing in task center :)